.github/workflows/contracts.yml

```
@@ -0,0 +1,333 @@
+name: contracts
+
+on: [ push ]
+
+env:
+  TOOLCHAIN_STABLE: 1.82.0 # toolchain for building contracts in stable channel
+  TOOLCHAIN_NIGHTLY: nightly-2024-09-01 # toolchain for building contracts in nightly channel for version 1.82.0
+  TOOLCHAIN_CHECK: 1.88.0 # toolchain for building cosmwasm-check tool
+
+jobs:
+  stable:
+    name: ${{ matrix.dir }}-${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        dir: [
+          burner,
+          crypto-verify,
+          cyberpunk,
+          empty,
+          hackatom,
+          ibc2,
+          ibc-callbacks,
+          ibc-reflect,
+          ibc-reflect-send,
+          nested-contracts,
+          queue,
+          reflect,
+          replier,
+          staking,
+          virus
+        ]
+    defaults:
+      run:
+        shell: bash
+        working-directory: contracts/${{ matrix.dir }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v5
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: ${{ env.TOOLCHAIN_STABLE }}
+          target: wasm32-unknown-unknown
```

Docs have only targets (plural). Please fix in all the other places.
```suggestion
          targets: wasm32-unknown-unknown
```